apps/dotcom/client/scripts/build.ts

```diff
@@ -95,14 +98,21 @@ async function build() {
 						dest: `${multiplayerServerUrl}$1`,
 						check: true,
 					},
-					// cache static assets immutably
 					{
 						src: '^/assets/(.*)$',
+						continue: true,
```

Comment:
it'd be good to note here what the `continue: true` here does.
we've scratched our heads on this config.json a couple times before so the more we can light the path here the better :)